### PR TITLE
9.0 write off analytic

### PR DESCRIPTION
**Pull request by vrenaville**
This add the possibility to set an analytic account, this analytic account  is propagated to the reconcile
to set this account analytic on the write-off move.

### CODE REVIEW

File: account_mass_reconcile/models/base_reconciliation.py [hunk 1]
```diff
@@ -159,10 +159,14 @@ def _reconcile_lines(self, lines, allow_partial=False):
...
             line_rs = ml_obj.browse(line_ids)
-            line_rs.reconcile(
-                writeoff_acc_id=writeoff_account,
-                writeoff_journal_id=self.journal_id
-                )
+            ctx = dict(self._context)
+            if self.analytic_account_id:
+                ctx.update(
+                    {'analytic_id': self.analytic_account_id.id})
+            line_rs.with_context(ctx).reconcile(
```
Comment (guewen) [inline account_mass_reconcile/models/base_reconciliation.py]:
```
if self.analytic_account_id:
    line_rs = line_rs.with_context(analytic_id=self.analytic_account_id.id)
line_rs.reconcile(
...
)
```

**Review by guewen — CHANGES_REQUESTED** [review]

**Comment by jgrandguillaume** [comment]
Hi,


Thanks, LGTM :+1: 

Regards,

Joël

**Review by leemannd — APPROVED** [review]
Code review only.
LGTM

**Comment by vrenaville** [comment]
@guewen 
Thanks for the review, I have applied the correction requested.

Vincent

**Review by guewen — APPROVED** [review]

**Pull request merged**